addon_service/external_service/link/models.py

```diff
@@ -10,19 +15,51 @@
 from addon_toolkit.interfaces.link import SupportedResourceTypes
 
 
+# TODO: need to trim down the features
+class LinkSupportedFeatures(Flag):
+    ADD_UPDATE_FILES = auto()
+    ADD_UPDATE_FILES_PARTIAL = auto()
+    DELETE_FILES = auto()
+    DELETE_FILES_PARTIAL = auto()
+    FORKING = auto()
+    LOGS = auto()
+    PERMISSIONS = auto()
+    REGISTERING = auto()
+    FILE_VERSIONS = auto()
+    COPY_INTO = auto()
+    DOWNLOAD_AS_ZIP = auto()
```

Comment:
@adlius , this my lazy copy from `StorageSupportedFeatures`. I wonder what we actually need for LINK. Is it decided by the CSV product provided?

[verified_link.csv](https://github.com/user-attachments/files/20459142/verified_link.csv)

Reply:
Yes. This should be determined by what products thinks is the common list of supported features for link addons. Also note that if there is a partially supported feature, we should add `<FEATURE_NAME>_PARTIAL` to the list, as that's how the FE would know if a feature is only partially supported.

Reply:
Thanks for the explanation. I assume `FALSE` means the feature should not be listed. So it should be like this

```python
FORKING_PARTIAL = auto()
LOGS_PARTIAL = auto()
PERMISSIONS = auto()
```